### DISCUSSION TITLE
No bans with Fail2ban yet, how to check it's working properly?

### DISCUSSION THREAD

**Discussion by vitobotta** [Mailu/Mailu#3111] [User support]
Hi! I have been using Mailu for some weeks already and I absolutely love it! When I set it up I configured Fail2ban as described in the docs, but so far I haven't seen any IPs getting banned with those filters. Could it be because I also have a custom filter that bans any IPs that attempt a portscan? So they kinda get banned before even trying to bruteforce the mail stuff? I followed the docs correctly I think but is there a way to confirm that this is working as expected? Thanks!

**Comment by lucassith** [comment]
Yeah, I can see that something went wront way with fail2ban in Mailu.
I had version 1.9 and it used to ban a loooot of bots, now I see that something is clearly not working.

I followed the documentation. First of all I don't see any authorization logs in the journal.d from front when I fail to log in.

Then - admin fail2ban is listening for rate limit message, but if I disable rate limitting, well, it won't work. Why not listen on failed attempts?

**Comment by nextgens** [comment]
@vitobotta here are my stats:
```
root@a1-full:~# fail2ban-client status bad-auth
Status for the jail: bad-auth
|- Filter
|  |- Currently failed:	11
|  |- Total failed:	3126
|  `- Journal matches:	CONTAINER_TAG=mailu-admin
`- Actions
   |- Currently banned:	0
   |- Total banned:	10
   `- Banned IP list:	
root@a1-full:~# fail2ban-client status bad-auth-bots
Status for the jail: bad-auth-bots
|- Filter
|  |- Currently failed:	4
|  |- Total failed:	73313
|  `- Journal matches:	CONTAINER_TAG=mailu-front
`- Actions
   |- Currently banned:	4
   |- Total banned:	114
   `- Banned IP list:	80.94.95.170 193.222.96.141 194.55.186.114 45.129.14.126
```

**Comment by vitobotta** [comment]
I checked now again and it seems to have banned some IPs in the meantime. Not many but I guess it's an indicator that it's working at least.
